src/components/MainActionButton.tsx

```diff
@@ -12,6 +12,7 @@ export interface mainActionButtonInterface {
 	children: React.ReactNode;
 	onClick: () => void;
 	variant?: Variant;
+	disabled?: boolean;
```

Comment:
There will be styling for this and I created a separate ticket. Essentially we will want to make the button look unclickable, but for now, it will still do nothing